include/seqan/basic/fundamental_concepts.h

```diff
@@ -299,6 +300,33 @@ SEQAN_CONCEPT(CopyConstructible,(T))
     T b;
 };
 
+/*!
+ * @concept MoveConstructibleConcept
+ * @brief A type with a move-constructor.
+ *
+ * @headerfile <seqan/basic.h>
+ *
+ * @signature MoveConstructible<T>
+ *
+ * @section Valid Expressions
+ *
+ * @code{.cpp}
+ * T a(rv);  // rv is an rvalue expression of type T
+ * @endcode
+ *
+ * @see DefaultConstructibleConcept
+ * @see CopyConstructibleConcept
+ */
+
+SEQAN_CONCEPT(MoveConstructible,(T))
+{
+    SEQAN_CONCEPT_USAGE(MoveConstructible)
+    {
+        T a{T{}};               // require move constructor
```

Comment:
this will be satisfied by the copy constructor, too

Reply:
Yes, which is a valid definition of the MoveConstructible concept.
See http://en.cppreference.com/w/cpp/concept/MoveConstructible